middleware/getVersionOfQuestionnaire.test.js

```diff
@@ -0,0 +1,98 @@
+const { getVersionOfQuestionnaire } = require("./index");
+
+describe("Getting a version of a questionnaire", () => {
+  let mockRequest, mockResponse, model;
+
+  beforeEach(() => {
+    mockRequest = data => {
+      return {
+        params: data
+      };
+    };
+
+    mockResponse = () => {
+      const res = {};
+      res.status = jest.fn().mockReturnValue(res);
+      res.json = jest.fn().mockReturnValue(res);
+      return res;
+    };
```

Comment:
I dont think these need to be set before every test. No risk of them being mutated in any of the tests.